dbms/src/Flash/Coprocessor/TiDBTime.h

```diff
@@ -21,6 +21,7 @@
 
 #include <Common/MyTime.h>
 #include <Core/Types.h>
+#include <Storages/Transaction/TiDB.h>
```

Comment:
Why add this?

Reply:
Because it actually depends on TiDB.h. The problem doesn't expose before, I think is probably the two head files are added together.